test-clients/src/main/java/com/hedera/services/bdd/suites/perf/AccountBalancesClientSaveLoadTest.java

```diff
@@ -200,18 +211,22 @@ private HapiApiSpec runAccountBalancesClientSaveLoadTest() {
 			@Override
 			public Optional<HapiSpecOperation> get() {
 				int next;
-				if ((next = createdSofar.getAndIncrement()) >= totalAccounts) {
+				if ((next = createdSofar.getAndDecrement()) < 0) {
 					return Optional.empty();
 				}
 
 				var op =  cryptoCreate(String.format("%s%s",ACCT_NAME_PREFIX , next))
-						.balance((long)(r.nextInt((int)ONE_HBAR) * 1000 + MIN_ACCOUNT_BALANCE))
+						.balance((long)(r.nextInt((int)ONE_HBAR) + MIN_ACCOUNT_BALANCE))
```

Comment:
Instead of randomization, why don't we just simply use some function of next?

Reply:
I'd like it to be spread out as much as possible as these values will be validated later one by one from both server side and client side.

Reply:
Unless you want randomization, I think you can find a function of `next` to satisfy the spread-out purpose. Besides, a deterministic function is much easier to validate later because we don't need to read/get the value somewhere before validating it.

Reply:
In general, a deterministic function (like transfer balance equals 1000 * next % ONE_HUNDRED_HBAR) will be easier to verify in most cases. 
But here, it's a little tricky. For this test, the design is that there is a period of random transfers from one account to another account picked from these accounts randomly. At the end, these accounts recorded from server side and client side should match one by one exactly (their hbar balance, tokens and token balances).  That's why, it's still impossible to predict the behaviors even we give it a deterministic function for transfer amount.